### PR TITLE
Add the last updated date underneath the edit button

### LINKED ISSUE #282: Add last edit info to pages
There is currently no indication on when a page was last edited.

It would be nice to extract this information directly using the git history and display it close to the edit button.

### PR DESCRIPTION
**Pull request by fsimonis**
I tried variations of how to format this for it to become compact:

```
Updated
Editted
Last edit
Last update
```

Last edit and update lead to a line break

Solves #282

### CODE REVIEW

**Comment by uekerman** [comment]
Could you paste a screenshot please?

**Comment by fsimonis** [comment]
Size | Preview
--- | ---
Large | image
Medium | image
Small | image

File: _layouts/page.html [hunk 1]
```diff
@@ -66,6 +66,9 @@ <h1 class="post-title-main">{{ page.title }}</h1>
                     <a target="_blank" rel="noopener" href="{{site.github_editme_path}}{{page.path}}" class="btn btn-default githubEditButton" role="button"><i class="fab fa-github fa-lg"></i> Edit me</a>
                 </div>
             {% endif %}
+            <div class="text-right">
+              <small>Updated {{ page.last_modified_at | date:  '%d %b %y' -}}</small>
```
Comment (MakisH) [inline _layouts/page.html:70]: How//where is this determined? Based on what?

**Review by MakisH — APPROVED** [review]
The renderings in the screenshots look very nice! I like the formulation, as well as that it nicely fits the width of the button.

File: _layouts/page.html (hunk 1 above)
Comment (fsimonis) [reply _layouts/page.html:70]: > This plugin determines a page's last modified date by checking the last Git commit date of source files. In the event Git is not available, the file's mtime is used.

https://github.com/gjtorikian/jekyll-last-modified-at?tab=readme-ov-file#last-modified-at-plugin

**Comment by uekerman** [comment]
Currently, all(?) pages show "Updated 03 May 24"?

**Comment by fsimonis** [comment]
On it

**Comment by fsimonis** [comment]
Fixed. We need a deep clone for the website now.

**Comment by MakisH** [comment]
We still need the same for the imported submodules, though. The tutorials pages, for example, also all show May 3 as last edit.

**Comment by fsimonis** [comment]
https://github.com/actions/checkout/issues/1124

Looks like we have to manually git clone

**Comment by fsimonis** [comment]
Opened an issue #404

**Pull request merged**